src/test/kotlin/org/dif/fixtures/Fixtures.kt

```diff
@@ -351,6 +660,53 @@ class JWE {
                 )
             )
         )
+
+        val BOB_DAMAGED_MESSAGE = """
```

Comment:
I would split Fixtures classes into multiple:
- test vectors for signed message
- test vectors for anoncrypt message
- test vectors for authcrypt message
- attachments data
- negative tests
etc.